vpr/src/route/connection_router.cpp

```diff
@@ -39,9 +39,12 @@ t_heap* ConnectionRouter<Heap>::timing_driven_route_connection_common_setup(
     int sink_node,
     const t_conn_cost_params cost_params,
     t_bb bounding_box) {
+    // Determine whether to use RCV data structures and functions or not
+    bool run_rcv = cost_params.delay_budget && cost_params.delay_budget->routing_budgets_algorithm == YOYO;
+
     //Re-add route nodes from the existing route tree to the heap.
     //They need to be repushed onto the heap since each node's cost is target specific.
-    add_route_tree_to_heap(rt_root, sink_node, cost_params);
+    add_route_tree_to_heap(rt_root, sink_node, cost_params, run_rcv);
```

Comment:
Is `run_rcv` constant for the entire duration of the `ConnectionRouter`'s lifetime?  If so, why not just make it a class variable, rather than passing it around?

Reply:
Good point yeah